File: app/src/main/java/com/example/flightstats/data/Airport.java
package com.example.flightstats.data;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "airports")
public class Airport {
    @PrimaryKey
    @NonNull
    public String iata = ""; // e.g. "AMS"

    public double lat;
    public double lng;
    public String country;  // ISO 2-letter e.g. "NL"
    public String name;     // "Amsterdam Airport Schiphol"
    public String city;     // "Amsterdam"
}

File: app/src/main/java/com/example/flightstats/data/Airport.kt
package com.example.flightstats.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "airports")
data class Airport(
    @PrimaryKey
    val iata: String, // e.g. "AMS"
    val lat: Double,
    val lng: Double,
    val country: String?,  // ISO 2-letter e.g. "NL"
    val name: String?,     // "Amsterdam Airport Schiphol"
    val city: String?      // "Amsterdam"
)
